### ISSUE TITLE
Keyboard navigation of an event calendar

### ISSUE THREAD

**Issue by LaurentGoderre** [wet-boew/wet-boew#6697]
@wet-boew/access-working-group

While redoing the event calendar I am running in a bit of an issue with regards to the keyboard navigation. The way it's working currently concerns me because of it's unpredictability. How do you guys think the navigation in the calendar should occur?

Should the keyboard user be able to go to any date even if there is no event associated, should we used tabbing instead arrow navigation or should we use modified arrow navigation?

**Comment by SBleeks** [comment]
The Outlook Calendar works well if you’re looking for a good example.
RightArrow to move to next day.
Left arrow to move to previous day.
UpArrow to move to previous week.
DownArrow to move to next week.

It is ideal to have the days (Monday, Tuesday,. Wednesday etc. Spoken as well.

Ideally a search ffunction can be used to pinpoint a specific day/month/year.

**Comment by jrowsell** [comment]
I agree with this, arrow navigation may be more intuitive.
Yes, the user should be able to access any date even if there is no event.

**Comment by LaurentGoderre** [comment]
This would only have worked if navigating to days without events is allowed. If the concesus is that it should, then it's a non issue.

**Comment by LaurentGoderre** [comment]
Thanks guys for the feedback! It's very helpful!

**Comment by hsrudnicki** [comment]
Can this issue be closed?

**Comment by hsrudnicki** [comment]
Thank you for keeping this issue's status updated, and continue whether this issue can be closed in v.4.0.19... or if otherwise?

**Comment by LaurentGoderre** [comment]
ANother question....what key should I use to navigate to the list of events? space and enter?

**Comment by masterbee** [comment]
@LaurentGoderre - let us try to nudge the access-working-group as well since I am interested to see what global recommendation can they make in terms of UI navigation in elements since there this will help us keep the development moving in terms of enhancements and fixes. Thanks for staying on this for this long.
